test/core/network/CMakeLists.txt

```diff
@@ -3,20 +3,20 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-addtest(peer_client_test
-    peer_client_test.cpp
-    )
-target_link_libraries(peer_client_test
-    peer_client
-    testutil_read_writer_helper
-    )
-
-addtest(peer_server_test
-    peer_server_test.cpp
-    )
-target_link_libraries(peer_server_test
-    peer_server
-    testutil_read_writer_helper
-    )
+#addtest(peer_client_test
```

Comment:
What is your intention in regard to this commented out code? Pls correct or remove

Reply:
The tests will be implemented, when the implementation is approved